packages/app/src/queries/rates/useCandlesticksQuery.ts

```diff
@@ -18,8 +19,14 @@ export const requestCandlesticks = async (
 ) => {
 	const ratesEndpoint = getRatesEndpoint(networkId)
 	const pythTvEndpoint = DEFAULT_PYTH_TV_ENDPOINT
-	const metalAssets = ['XAU', 'XAG']
-	const prefix = metalAssets.includes(currencyKey!) ? 'Metal' : 'Crypto'
+	const nonCryptoTypes: AssetTypes = {
```

Comment:
I think we can lift this out of the component and maybe rename as `NON_CRYPTO_ASSET_TYPES`

Reply:
Good idea. Updated.